### PR TITLE
Migrate BlueConfig keys to SONATA simulation config

### PR DESCRIPTION
**Pull request by WeinaJi**
* Remove forward_skip from SONATA (BBPBGLIB-913)
* Remove minis_single_vesicle from SONATA (BBPBGLIB-915)
* New keys for stimulus_seed, ionchannel_seed, minis_seed, synapse_seed (BBPBGLIB-920)
* Remark on the weights of replay synapses (BBPBGLIB-922)
* Add `series_resistance` in seclamp (BBPBGLIB-921)

### CODE REVIEW

File: source/sonata_simulation.rst [hunk 1]
```diff
@@ -281,6 +281,7 @@ Cells are held at indicated membrane voltage by injecting adapting current.
    Property                       Type       Requirement  Description
    ============================== ========== ============ ==========================================
    voltage                        float      Mandatory    Specifies the membrane voltage the targeted cells should be held at in mV.
+   rs                             float      Optional     Specifies the series resistance in Mohm. Default is 0.01 Mohm.
```
Comment (matz-e) [inline source/sonata_simulation.rst:284]:
```suggestion
   rs                             float      Optional     Specifies the series resistance in MΩ. Default is 0.01 MΩ.
```

File: source/sonata_simulation.rst [hunk 2]
```diff
@@ -280,6 +281,7 @@ Cells are held at indicated membrane voltage by injecting adapting current.
    Property                       Type       Requirement  Description
    ============================== ========== ============ ==========================================
    voltage                        float      Mandatory    Specifies the membrane voltage the targeted cells should be held at in mV.
+   rs                             float      Optional     Specifies the series resistance in M :math:`\Omega`. Default is 0.01 M :math:`\Omega`.
```
Comment (mgeplf) [inline source/sonata_simulation.rst:284]: would it make sense to call this `resistance` or `series_resistance`?

File: source/sonata_simulation.rst [hunk 3]
```diff
@@ -82,7 +82,10 @@ Parameters defining global simulation settings. As NEURON is the engine used for
...
    integration_method              text       Optional    Selects the NEURON/CoreNEURON `integration method <https://nrn.readthedocs.io/en/latest/python/simctrl/programmatic.html#secondorder>`_. This parameter sets the NEURON global variable h.secondorder. The allowed values are ‘0’ (default) for fully implicit backward euler, ‘1’ for Crank-Nicolson and ‘2’ for Crank-Nicolson with fixed ion currents.
-   forward_skip                    integer    Optional    When included, then run without Stimulus or Reports for given duration prior to t=0 using a timestep computed as dt=0.1*forward_skip. This is to get the cells quickly past any initial transience. Specified in ms.
+   stimulus_seed                   integer    Optional    A non-negative integer used for seeding noise stimuli and any other future stochastic stimuli, default is 0.
+   ionchannel_seed                 integer    Optional    A non-negative integer used for seeding stochastic ion channels, default is 0.
+   minis_seed                      integer    Optional    A non-negative integer used for seeding the Poisson processes that drives the minis, default is 0.
```
Comment (seirios) [inline source/sonata_simulation.rst:87]:
```suggestion
   minis_seed                      integer    Optional    A non-negative integer used for seeding the Poisson processes driving the minis, default is 0.
```

File: source/sonata_simulation.rst (hunk 2 above)
Comment (seirios) [reply source/sonata_simulation.rst:284]: `series_resistance` would definitely be more human-readable

File: source/blueconfig.rst [hunk 4]
```diff
@@ -265,6 +265,34 @@ Neurodamus.
          For random sequences, the BaseSeed is added in order to give the user
          the capacity to change the sequences.
 
+    .. blueconfig_value:: StimulusSeed
+        :type: int
+        :required: False
+        :unit:
+        :description:
+         A non-negative integer used for seeding noise stimuli and any other future stochastic stimuli.
```
Comment (seirios) [inline source/blueconfig.rst:273]:
```suggestion
         A non-negative integer used for seeding noise stimuli and any other future stochastic stimuli. Default is 0.
```

File: source/blueconfig.rst [hunk 5]
```diff
@@ -265,6 +265,34 @@ Neurodamus.
...
+
+    .. blueconfig_value:: IonChannelSeed
+        :type: int
+        :required: False
+        :unit:
+        :description:
+         A non-negative integer used for seeding stochastic ion channels.
```
Comment (seirios) [inline source/blueconfig.rst:280]:
```suggestion
         A non-negative integer used for seeding stochastic ion channels. Default is 0.
```

File: source/blueconfig.rst [hunk 6]
```diff
@@ -265,6 +265,34 @@ Neurodamus.
...
+        :unit:
+        :description:
+         A non-negative integer used for seeding the Poisson processes that drives the minis.
+
+    .. blueconfig_value:: SynapseSeed
+        :type: int
+        :required: False
+        :unit:
+        :description:
+         A non-negative integer used for seeding stochastic synapses.
```
Comment (seirios) [inline source/blueconfig.rst:294]:
```suggestion
         A non-negative integer used for seeding stochastic synapses. Default is 0.
```

File: source/sonata_simulation.rst [hunk 7]
```diff
@@ -281,7 +281,7 @@ Cells are held at indicated membrane voltage by injecting adapting current.
    Property                       Type       Requirement  Description
    ============================== ========== ============ ==========================================
    voltage                        float      Mandatory    Specifies the membrane voltage the targeted cells should be held at in mV.
-   rs                             float      Optional     Specifies the series resistance in M :math:`\Omega`. Default is 0.01 M :math:`\Omega`.
+   rs                             float      Optional     Specifies the series_resistance in M :math:`\Omega`. Default is 0.01 M :math:`\Omega`.
```
Comment (seirios) [inline source/sonata_simulation.rst]:
```suggestion
   series_resistance              float      Optional     Specifies the series resistance in M :math:`\Omega`. Default is 0.01 M :math:`\Omega`.
```

Comment (WeinaJi) [reply source/sonata_simulation.rst]: Yes, the last change that I've pushed has this change in.

**Review by seirios — APPROVED** [review]
LGTM

**Pull request merged**